### DISCUSSION TITLE
Where to start

### DISCUSSION THREAD

**Discussion by lcpt** [xcfem/xc#96] [General]
# :sparkles: **Take a look to the document with updated content [here](https://github.com/xcfem/xc/blob/master/doc/how_to_start.md).**

XC is becoming popular and some of you have asked about how to start using it. I've compiled some documents which can help:

### Examples
- [XC examples](https://github.com/xcfem/xc_examples)

### Tutorials
- [Getting started with XC. Axial force in a bar subjectedto a uniform temperature increase.](https://github.com/xcfem/xc_examples/blob/master/XC_tutorial_001/tutorial001_truss_temp.pdf)
- [Analysis of a vibrating string under tension.](https://raw.githubusercontent.com/xcfem/xc_examples/master/XC_tutorial_002/tutorial002_eigen_vibr_string.pdf)
- [Properties of a reinforced concrete sectionobtained from a fiber-section model](https://raw.githubusercontent.com/xcfem/xc_examples/master/XC_tutorial_003/tutorial003_fiber_section.pdf)

### Verification tests
- [XC verification tests](https://github.com/xcfem/xc/tree/master/verif/tests)

### Git for beginners

- [Git for beginners](https://product.hubspot.com/blog/git-and-github-tutorial-for-beginners)
- [Git para principiantes](https://www.ionos.es/digitalguide/paginas-web/desarrollo-web/tutorial-de-git/)

### How to create a fork

- [Fork a repo](https://docs.github.com/en/github/getting-started-with-github/quickstart/fork-a-repo)
- [Bifurcar un repositorio](https://docs.github.com/es/github/getting-started-with-github/quickstart/fork-a-repo)

## Python

- [Python for beginners](https://www.python.org/about/gettingstarted/)
- [Python para principiantes](https://www.ionos.es/digitalguide/paginas-web/desarrollo-web/tutorial-de-python/)

## C++

- [A Tour of C++ (C++ In-Depth Series) 1st Edition](https://geni.us/6fnnXpc) Bjarne Stroustrup. For the brave and the impatient :)
- [C++ Primer 5th Edition](https://geni.us/Y6Udqa).
- [C++ Pocket Reference 1st Edition](https://geni.us/w3tnB9).

## Finite element analysis

- [The Finite Element Method: Its Basis and Fundamentals](https://www.elsevier.com/books/the-finite-element-method-its-basis-and-fundamentals/zienkiewicz/978-1-85617-633-0)
- [OpenSees tutorial](https://opensees.berkeley.edu/workshop/NEESgridUser-EndWorkshop2005_presentations/NEESgridUser-EndWorkshopOpenSeesTutorial_SilviaMazzoni2005.pdf)

**Comment by lcpt** [comment]
I've put this information [here](https://github.com/xcfem/xc/blob/master/doc/how_to_start.md). I'll try to update this file from time to time.

**Comment by ebrahimraeyat** [comment]
Hello @lcpt. Thank you so much for your information.

**Reply by lcpt** [reply]
If you have any suggestions, feel free to write them here.
